src/main/java/de/tum/in/www1/artemis/web/rest/OrganizationResource.java

```diff
@@ -104,7 +107,7 @@ public OrganizationResource(OrganizationService organizationService, Organizatio
     }
 
     /**
-     * DELETE /organizations/user/:userLogin/organization/:organizationId :
+     * DELETE organizations/{organizationId}/users/{userLogin} :
```

Comment:
Why do we suddenly use braces here? This does not seem intentional.

Reply:
Agree
```suggestion
     * DELETE organizations/:organizationId/users/:userLogin :
```